examples/quickstart/jupyter-notebooks/notebooks/04-api/01-delete-api-tutorial.ipynb

```diff
@@ -0,0 +1,975 @@
+   "metadata": {},
+   "source": [
+    "Next, send a `POST` with the payload `{\"interval\": \"2015-09-12T00:00:00.000Z/2015-09-13T01:00:00.000Z\"}` to mark the entirety of the table as \"unused.\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25639752",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql_request = {\n",
+    "  \"interval\": \"2015-09-12T00:00:00.000Z/2015-09-13T01:00:00.000Z\"\n",
+    "}\n",
+    "\n",
+    "response = session.post(endpoint, json=sql_request);\n",
+    "\n",
+    "response.status_code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbbba823",
+   "metadata": {},
+   "source": [
+    "To verify the segment changes, the following cell sets the endpoint to `/druid/v2/sql` and send a SQL-based request. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eac1db4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = druid_host + '/druid/v2/sql'\n",
+    "sql_request = {\n",
+    "  \"query\": \"SELECT segment_id FROM sys.segments WHERE \\\"datasource\\\" = 'wikipedia_hour'\",\n",
+    "  \"resultFormat\": \"objectLines\"\n",
+    "}\n",
+    "\n",
+    "response = session.post(endpoint, json=sql_request);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deae8727",
+   "metadata": {},
+   "source": [
+    "Run the next cells to view the response. You should see that the `response.text` returns nothing, but `response.status_code` returns a 200. \n",
+    "\n",
+    "The response should return the remaining segments, but since the table was deleted, there are no segments to return."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12c11291",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response.text\n",
+    "response.status_code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e9a74da",
+   "metadata": {},
+   "source": [
+    "So far, you've soft deleted the table. Run the following cells to permanently delete the table from deep storage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6c3d7ec9",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = druid_host + '/druid/coordinator/v1/datasources/wikipedia_hour/intervals/2015-09-12_2015-09-13'\n",
+    "endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98834167",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = session.delete(endpoint);\n",
+    "response.status_code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b2d59c8",
+   "metadata": {},
+   "source": [
+    "## Delete by segment ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4e8453e",
+   "metadata": {},
+   "source": [
+    "In addition to deleting by interval, you can delete segments by using `segment_id`. Let's load in some new data to work with.\n",
```

Comment:
Is there a reason this comes after we delete the whole table? it seems inefficient to have them have to reingest a table

Reply:
I grouped deleting by interval and table together since they both use the same base method, passing a time interval.